### ISSUE TITLE
Don't use this slider

### ISSUE THREAD

**Issue by 1SolPi1** [xwpongithub/vue-range-slider#11]
Don't use this slider newer!!!!!!!!!!!!!!!!!

**Comment by aiman1717a** [comment]
Why? whats the reason?

**Comment by 1SolPi1** [comment]
> Why? whats the reason?

Slider blocks all input on page

Look:

https://github.com/xwpongithub/vue-range-slider/issues/15
https://github.com/xwpongithub/vue-range-slider/issues/13
https://github.com/xwpongithub/vue-range-slider/issues/7
https://github.com/xwpongithub/vue-range-slider/issues/4

This pull from 28 oct 2019 
https://github.com/xwpongithub/vue-range-slider/pull/5

**Comment by davidbonting** [comment]
True, this slider gives really weird side effects to the rest of the code. I think the best alternative is this: https://github.com/xwpongithub/vue-range-slider/issues/13#issuecomment-628857746
